### DISCUSSION TITLE
How to interpret plot.variable() for survival forest?

### DISCUSSION THREAD

**Discussion by arbet003** [kogalur/randomForestSRC#121] [Q&A] [answered]
For survival forests, `plot.variable()`  argument `surv.type` has 3 types: 

* Mortality (`mort`)
* Relative frequency of mortality (`rel.freq`).
* Predicted survival (`surv`) at a specific time point

When plotting a continuous variable, how do you interpret the y-axis for `mort` and `rel.freq`?  

For example:

```
library("randomForestSRC")
data(veteran, package = "randomForestSRC")
v.obj <- rfsrc(Surv(time,status)~., veteran, ntree = 100)

plot.variable(v.obj, c("karno"), surv.type = "mort")
```

**1) what does a mortality value of 100 mean?**

![mort_plot_example](https://user-images.githubusercontent.com/16568477/142341133-2b6269f2-0c4a-4c66-97bb-8832d733c546.PNG)

**2) what does a standardized mortality value of 0.6 mean?**

`plot.variable(v.obj, c("karno"), surv.type = "rel.freq")
`
![rel_freq_plot_example](https://user-images.githubusercontent.com/16568477/142341265-37eb273d-4cb7-40ce-9f9f-719441a594f0.PNG)

**Comment by ishwaran** [comment, marked as answer]
This value is described in the help file for rfsrc:

**Values returned depend heavily on the family.  In particular  predicted values from the forest (‘predicted’ and ‘predicted.oob’) are as follows:**

 **3. For survival, a vector of mortality values (Ishwaran et al., 2008) representing estimated risk for each individual calibrated to the scale of the number of events (as a specific example, if _i_ has a mortality value of 100, then if all individuals had the same x-values as _i_, we would expect an average of 100 events).**

The relative mortality divides this value by max(n, max(mortality)) to put this on a scale from 0 to 1.

I know that help files can sometimes be difficult to navigate, so you might find our webpage helpful for this kind of information, https://luminwin.github.io/randomForestSRC
